### PR TITLE
Typo fixes, more detail, bug fixes in docs

### PR DESCRIPTION
**Pull request by benlk**
## Changes

- Comments on https://github.com/INN/super-cool-ad-inserter-plugin/pull/21/files
- Better explanation of how things work with regards to "Will all my ads be displayed?"

### CODE REVIEW

**Comment by benlk** [comment]
Tossing to Adam for more line edits.

@jackbrighton, are all these understandable?

File: docs/display-settings.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+The Super Cool Ad Inserter provides lots of flexibility in how and where ads are placed within posts. By default, 2 ads will be inserted automatically in a post, beginning 3 paragraphs after the beginning and every 3 paragraphs after that. They will not appear if the post is shorter than 6 paragraphs. You can easily change these default settings for ad automatic insertion in the [Ad Inserter Plugin Options panel](configuration.md).
 
 ## Prevent Automatic Insertion of Ads in a Post
 
-By default once [configured](configuration.md) SCAIP will insert ads in every post. But you may have posts for which you want to exclude advertising. This is simple to do using the Super Cool Ad inserter post meta box that appears in the post editing page:
+By defaulti, once [configured](configuration.md) SCAIP will insert ads in every post. But you may have posts for which you want to exclude advertising. This is simple to do using the Super Cool Ad inserter post meta box that appears in the post editing page:
```
Comment (jackbrighton) [inline docs/display-settings.md]: defaulti should of course be default.

File: docs/display-settings.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 ![Super Cool Ad inserter post meta box](./img/scaip-post-meta-box.png)
 
-You can check the box to "prevent automatic addition of ads to this post," or alternatively you can add the shortcode `[ad no]` at the start of the story. In either case, no ads will appear in the post.
+You can check the box to "prevent automatic addition of ads to this post" or add the shortcode `[ad no]` at the start of the story. In either case, no ads will appear in the post. If you uninstall the Super Cool Ad Inserter Plugin, the `[ad no]` shortcodes will remain in your posts, so it is recommended to use the checkbox instead of `[ad no]`.
```
Comment (jackbrighton) [inline docs/display-settings.md]: oh great catch!

File: docs/display-settings.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ You can also reorder the display of Inserted Ad Positions by placing shortcodes
 
 ## Ad Spacing Using Empty Inserted Ad Positions
 
-You can add Text Widgets to each and every numbered Inserted Ad Position, and they will display every nth paragraph based on the number of paragraphs you defined in **Plugins** > **Ad Inserter**  [options](configuration.md). 
+You can add Text Widgets to each and every numbered Inserted Ad Position, and they will display every nth paragraph based on the number of paragraphs you defined in **Plugins** > **Ad Inserter**  [options](configuration.md).
```
Comment (jackbrighton) [inline docs/display-settings.md]: Should this say "You can add Widgets" instead of "You can add Text Widgets"? Since you can add other kinds of widgets.

**Pull request merged**